src/app/developer/components/utils.ts

```diff
@@ -1,6 +1,41 @@
+import { BOOKS } from '../../../config';
 import { Book } from '../../content/types';
-import { formatBookData } from '../../content/utils';
+import { formatBookData, getUrlParamForPageId } from '../../content/utils';
 import { AppServices } from '../../types';
+import { assertDefined } from '../../utils';
+
+// from https://docs.google.com/spreadsheets/d/1Hj5vm2AbEiLgxgcbNRi550InzFTLvBOv16-Fw6P-c5E/edit#gid=2066350488
+export const CANONICAL_MAP: ObjectLiteral<string[] | undefined> = {
+  /* Algebra & Trigonometry */ '13ac107a-f15f-49d2-97e8-60ab2e3b519c' : [
+    /* College Algebra */ '9b08c294-057f-4201-9f48-5d6ad992740d',
+    /* Precalculus */ 'fd53eae1-fa23-47c7-bb1b-972349835c3c',
```

Comment:
maybe we should break down and have *one* file that just maps book names to uids so we dont have to have comments all over the place